.travis.yml

```diff
@@ -14,8 +14,7 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-mono:
-  - 4.0.5
+mono: none
```

Comment:
It was actual honest question, but always happy to chat :smile: Plus I was going to add that I am not sure how much value running on Travis on Mono could add anyway, since it is possible to work on Mono and still be broken on Xamarin. In any case, I would like to hear what @bricelam thinks.